### DISCUSSION TITLE
Debugging random freezings

### DISCUSSION THREAD

**Discussion by avidseeker** [gokcehan/lf#1768] [General]
lf.log is not verbose enough and I don't know how to replicate the freezing (it's random).

```
2024/07/03 00:20:32 hi!
2024/07/03 00:20:32 reading file: /home/user/.config/lf/icons
2024/07/03 00:20:32 loading files: []
2024/07/03 00:20:32 reading file: /home/user/.config/lf/lfrc
2024/07/03 00:20:32 loading files: []
2024/07/03 00:20:33 pushing keys :find<enter>r
2024/07/03 00:20:33 command: find
2024/07/03 00:20:33 shell: ${{ case ${f##*.} in ... }} -- []
```
any way to increase verbosity and debug this?

**Comment by CatsDeservePets** [comment]
There is no such setting unfortunately. You could add more log statements and compile `lf` yourself for local testing (Go is not that hard so this is doable).
